src/@types/map.d.ts

```diff
@@ -0,0 +1,17 @@
+interface MapProps {
+    data: any;
```

Comment:
The `any` here should be a geojson type. There exists https://www.npmjs.com/package/@types/geojson but I haven't tested or looked into it at all yet

Reply:
I trust the way this library's @types does GeoJSON types if that's any help: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f7ec78508c6797e42f87a4390735bc2c650a1bfd/types/topojson-client/topojson-client-tests.ts